packages/pluggableWidgets/datagrid-web/src/ui/Datagrid.scss

```diff
@@ -184,6 +184,10 @@ $dragging-color-effect: rgba(10, 19, 37, 0.8);
             text-overflow: ellipsis;
             overflow: hidden;
         }
+
+        > .empty-placeholder {
+            width: 100%;
+        }
```

Comment:
A `div` [is a block element](https://developer.mozilla.org/en-US/docs/Web/HTML/Block-level_elements), so this width 100% CSS rule is redundant right? Or do we reset the style somewhere before?

Reply:
The parent is a display: flex but with flex-grow <> 1, which means it fits the content :)